### DISCUSSION TITLE
How to define DETECTIONS_PER_IMAGE in the lazy config?

### DISCUSSION THREAD

**Discussion by nechir123** [facebookresearch/detectron2#4384] [General]
I wanna export my model that has been trained by the lazyconfig_train_net.py. The default value for DETECTIONS_PER_IMAGE= 100 i assume. How can i set this via lazy? When the training was done in the normal way, i could just override the value cfg.TEST.DETECTIONS_PER_IMAGE = 1000.

**Comment by MichaelALee** [comment]
Hi nechir123,
I am also struggling with finding the correct way to override this with the new lazy configs.
Did you manage to find a solution?

**Reply by nechir123** [reply]
Hey, Sorry for late response. I havent been able to figure it out. @MichaelALee

**Reply by nechir123** [reply]
@ppwwyyxx Do you know the solution for this case?

**Reply by nechir123** [reply]
@MichaelALee I think i found the solution. You need to add this line to "detectron2/configs/common/models/mask_rcnn_fpn.py", and then you can override that parameter. 
![image](https://user-images.githubusercontent.com/24247356/184315428-b649955b-44e0-4a58-b115-d9742ce134b8.png)

**Reply by MichaelALee** [reply]
Great find! Thank you.

**Reply by obdwinston** [reply]
Thanks that worked for me~
